bucket/qbittorrent.json

```
@@ -16,14 +16,22 @@
             "hash": "72ecfe934a8dc819f47bb100011b9bb3569b4e9ee2d1fbe7cd1d048662b2198b"
         }
     },
-    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
```

Remove what you added and keep only persist and this:
```suggestion
    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Force -Recurse",
```
Check for persited folder is redundant